ee/codegen/src/serializers/vellum.ts

```diff
@@ -1571,6 +1658,7 @@ export const ConditionalNodeSerializer: ObjectSchema<
   base: CodeResourceDefinitionSerializer.optional(),
   definition: CodeResourceDefinitionSerializer.optional(),
   adornments: listSchema(AdornmentNodeSerializer).optional(),
+  ports: listSchema(NodePortSerializer).optional(),
```

Comment:
[oos] How does this work with conditional nodes? can we follow up with a test to ensure that we don't generate two node port classes here?